src/main/java/com/dynatrace/openkit/core/WebRequestTracerBaseImpl.java

```diff
@@ -64,31 +66,42 @@ public String getTag() {
 
     @Override
     public WebRequestTracer setResponseCode(int responseCode) {
-        this.responseCode = responseCode;
+        if (!isStopped()) {
```

Comment:
naming: above you used isActionOpen, here you use isStopped
I would use the same kind of logic troughout the classes. All negative or all positive.